packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -106,15 +115,16 @@ describe(`request-queue`, function() {
       expect(callbacks.requestWillEnqueue.calledOnce).to.be.true;
       expect(callbacks.requestWillEnqueue.calledWith(sinon.match.has('request')))
           .to.be.true;
-
+      expect(performance.now() - startTime).to.be.above(800);
```

Comment:
@prateekbh this will just introduce flakey tests. Please move 800 out into a static VARIABLE and please use startimers to force a 800ms step forward regardless of actual time.